<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JtpzDlg</class>
 <widget class="QDialog" name="JtpzDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,4">
     <item>
      <widget class="QGroupBox" name="gpSalary">
       <property name="title">
        <string>工资</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="chkSalary">
          <property name="text">
           <string>记提工资</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gpAdded">
       <property name="title">
        <string>增值税</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="chkAddedvaluetax">
          <property name="text">
           <string>记提增值税</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gpOther">
       <property name="title">
        <string>其他税金</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="chkCbj">
            <property name="text">
             <string>残保金</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkStampDuty">
            <property name="text">
             <string>印花税</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkCity">
            <property name="text">
             <string>城市维护建设税</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkEducation">
            <property name="text">
             <string>教育费附加</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>chkCbj</zorder>
       <zorder>chkStampDuty</zorder>
       <zorder>chkCity</zorder>
       <zorder>chkEducation</zorder>
       <zorder>gpSalary</zorder>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>创建</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>JtpzDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCreate</sender>
   <signal>clicked()</signal>
   <receiver>JtpzDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
